server/services/email-campaigns/inactive-workplaces/findParentWorkplaces.js

```diff
@@ -9,6 +9,9 @@ const buildWorkplaces = (workplaces) => {
   return workplaces.reduce((acc, workplace) => {
     if (workplace.IsParent) {
       acc[workplace.EstablishmentID] = {
+        ...{
+          subsidiaries: [],
+        },
```

Comment:
Why not just add subsidiaries to the object? 
```suggestion
      subsidiaries: [],
```